frontend/packages/ux-editor/src/components/config/RuleComponent.tsx

```diff
@@ -146,70 +148,62 @@ class Rule extends React.Component<IRuleComponentProps, any> {
           </div>
           {this.state.ruleConnection.selectedFunction ? (
             <>
-              <div className='form-group a-form-group mt-2'>
-                <h2 className='a-h4'>
+              <div>
+                <h2 className={classes.subTitle}>
                   {this.props.t('ux_editor.modal_configure_rules_configure_input_header')}
                 </h2>
                 {Object.keys(this.props.ruleModelElements[selectedMethodNr].inputs).map(
-                  (key: any, index: any) => {
+                  (key: any) => {
                     const paramName = key;
                     return (
-                      <div className='align-items-center mb-1 row' key={index}>
-                        <div className='col-3 col'>
-                          <div className='form-group a-form-group mt-1 disabled'>
-                            <label className='a-form-label' htmlFor={paramName}>
-                              {this.props.t(
-                                'ux_editor.modal_configure_rules_configure_input_param_helper'
-                              )}
-                            </label>
-                            <input
-                              id={paramName}
-                              name={paramName}
-                              type='text'
-                              className='form-control'
-                              value={this.props.ruleModelElements[selectedMethodNr].inputs[key]}
-                              width={10}
-                              disabled={true}
+                      <>
```

Comment:
Since this element is within a map (loop), then we need to set a key-identifier to the element. You can read more about it here: https://reactjs.org/docs/lists-and-keys.html. 

Without this key-prop, you should see an error message within the dev-tools console, _"Each child in a list should have a unique "key" prop."_

```suggestion
                      <React.Fragment key={key}>
```

Reply:
That's right, I had it just in line 165 `<div className={classes.configureInputParamsContainer} key={key}>` I have not noticed that map includes many html elements. It works good now with your solution. THANKS

Reply:
I did not notice that you had the key prop on line 165, you should remove it. The key should be on the first child element of the map method. Within this case, it is the `<React.Fragment>` element. :) 

Worth mentioning that it's important that element within the same loop does not have the same key, they should be unique identifiers for each element. :)